_api-reference/profile.md

```diff
@@ -979,51 +968,41 @@ The following is an example response for a concurrent search with three segment
-|`breakdown.avg_<method>_count`	| The average number of invocations of a `<method>` on any slice. Breakdown stats for the `create_weight` method do not include profiled `avg` count because the method runs at the query level rather than the slice level. |
+Field | Data type | Description
+:--- | :--- | :---
+`type` | String | The aggregator type. In the [non-global aggregation example response](#example-response-non-global-aggregation), the aggregator type is `AvgAggregator`. [Global aggregation example response](#example-request-global-aggregation) contains a `GlobalAggregator` with an `AvgAggregator` child.
+`description` | String | Contains a Lucene explanation of the aggregation. Helps differentiate aggregations with the same type.
+`time_in_nanos` | Long | The total elapsed time for this aggregation, in nanoseconds. For concurrent segment search, `time_in_nanos` is the total amount of time across all slices (the difference between the last completed slice execution end time and the first slice execution start time).	
+[`breakdown`](#the-breakdown-object-1) | Object | Contains timing statistics about low-level Lucene execution.
+`children` | Array of objects | If an aggregation has subaggregations (children), this field contains information about the subaggregations.
+`debug` | Object | Some aggregations return a `debug` object that describes the details of the underlying execution.
+`max_slice_time_in_nanos`	|The maximum amount of time taken by any slice to run an aggregation, in nanoseconds.	
+`min_slice_time_in_nanos`	|The minimum amount of time taken by any slice to run an aggregation, in nanoseconds.	
+`avg_slice_time_in_nanos`	|The average amount of time taken by any slice to run an aggregation, in nanoseconds.	
+`<method>`	|The total elapsed time across all slices (the difference between the last completed slice execution end time and the first slice execution start time). For example, for the `collect` method, it is the total time spent collecting documents into buckets across all slices.	
+`max_<method>`	|The maximum amount of time taken by any slice to run an aggregation method.	
+`min_<method>`|The minimum amount of time taken by any slice to run an aggregation method.	
+`avg_<method>`	|The average amount of time taken by any slice to run an aggregation method.	
+`<method>_count`	|The total method count across all slices. For example, for the `collect` method, it is the total number of invocations of this method needed to collect documents into buckets across all slices.	
+`max_<method>_count`	|The maximum number of invocations of a `<method>` on any slice.	
+`min_<method>_count`	|The minimum number of invocations of a `<method>` on any slice.	
+`avg_<method>_count`	|The average number of invocations of a `<method>` on any slice.	
```

Comment:
These are part of breakdown within aggregation section and not in the top level aggregation section.

Reply:
Yes, these method stats should go within "the aggregations array" section below.